browser/ui/views/tabs/brave_tab_search_button.cc

```diff
@@ -25,6 +25,9 @@ gfx::Size BraveTabSearchButton::CalculatePreferredSize() const {
 SkPath BraveTabSearchButton::GetBorderPath(const gfx::Point& origin,
                                            float scale,
                                            bool extend_to_top) const {
+  if (fill_color_disabled_)
+    return {};
```

Comment:
I'm confused as to what this part is achieving? It seems like we're aiming to disable the highlight, but it's still visible on the same way in master and this branch (to me!). Also I'm noticing the Collapse/Expand button has no highlight at all but that is a separate issue? Perhaps you could add a comment explaining why we wouldn't want the highlight, either here on in `vertical_tab_strip_region_view` where this is set?